rollout/trafficrouting/alb/alb.go

```diff
@@ -125,9 +128,17 @@ func (r *Reconciler) VerifyWeight(desiredWeight int32, additionalDestinations ..
 		r.cfg.Status.ALB = nil
 		return nil, nil
 	}
+
+	if !rolloututil.ShouldVerifyWeight(r.cfg.Rollout) {
```

Comment:
Could you leave a few comments on what we are actually doing here. Sorry it wasn't obvious to me.